documentation/existing.md

```diff
@@ -158,7 +218,25 @@ Below is summary of supported parameters in the `vsphere.conf` file
 
         datastore = "DatastoreStorageFolder/datastore1"
 
-## Add flags to controller-manager, API server and Kubelet
+### Step - 6: Add flags to controller-manager, API server and Kubelet
+
+#### For kubernetes version 1.9.x
+
+** Master node **
```

Comment:
I still see ** Master node ** in the documentation. Can you check it out.

Reply:
Done